eslint.shared.js

```diff
@@ -9,91 +9,100 @@ const {
   createTypeScriptOverride,
 } = require('eslint-config-galex/dist/overrides/typescript');
 
-const JS_RULES = {
```

Comment:
I'm reverting to inlining the rules objects in the `createConfig` call to get better type safety.